backend/src/lib.rs

```diff
@@ -266,20 +266,7 @@ async fn openapi() -> &'static str {
 }
 
 pub async fn shutdown_signal(tx: tokio::sync::broadcast::Sender<()>) -> anyhow::Result<()> {
-    use std::io;
-    use tokio::signal::unix::SignalKind;
-
-    async fn terminate() -> io::Result<()> {
-        tokio::signal::unix::signal(SignalKind::terminate())?
-            .recv()
-            .await;
-        Ok(())
-    }
-
-    tokio::select! {
-        _ = terminate() => {},
-        _ = tokio::signal::ctrl_c() => {},
-    }
+    tokio::signal::ctrl_c().await.unwrap();
```

Comment:
why can we remove the terminate signal listener ? Is that already handled by ctrl_c ?

Reply:
I believe so. The [docs](https://docs.rs/tokio/latest/tokio/signal/fn.ctrl_c.html) say
> Even if this Signal instance is dropped, subsequent SIGINT deliveries will end up captured by Tokio, and the default platform behavior will NOT be reset.

Which to me reads it'll just detect SIGINT

Reply:
SIGINT yes, this is what ctrl+c is, but what about sigterm ?

Reply:
<https://github.com/tokio-rs/tokio/blob/a03e0420249d1740668f608a5a16f1fa614be2c7/tokio/src/signal/unix.rs#L496> doesn't look like it. Will change to also handle sigterm